forum/qa-include/qa-theme-base.php

```diff
@@ -290,8 +290,13 @@ public function head_links()
 	public function head_script()
 	{
 		if (isset($this->content['script'])) {
-			foreach ($this->content['script'] as $scriptline)
+		    $deferredScripts = ['ckeditor.js?'];
+
+			foreach ($this->content['script'] as $scriptline) {
+                $scriptline = $this->tryDeferScript($deferredScripts, $scriptline);
+
 				$this->output_raw($scriptline);
+            }

```

Comment:
formatowanie do poprawy. Poza tym nie podoba mi się pomysł edycji czegokolwiek bezpośrednio w kodzie core.

Reply:
Z chęcią przygarnę lepszy sposób na edycję tego niż bezpośrednio w core. :)

Reply:
Jak tu stoimy, jakiś pomysł na to? Jak nie w core to co, plugin na zmianę tych kilku linii z dodaniem jednego skryptu?